pandas/tests/arrays/string_/test_string.py

```diff
@@ -269,3 +267,26 @@ def test_value_counts_na():
     result = arr.value_counts(dropna=True)
     expected = pd.Series([2, 1], index=["a", "b"], dtype="Int64")
     tm.assert_series_equal(result, expected)
+
+
+@pytest.mark.parametrize("func", ["min", "max"])
+@pytest.mark.parametrize("skipna", [True, False])
+def test_reduction(func, skipna):
```

Comment:
can you test a series with nulls as well

Reply:
See the test below